boxoffice/static/js/models/item_collection.js

```diff
@@ -1,8 +1,15 @@
 import {Util, fetch} from './util.js';
+import {IndexModel} from './index.js';
 
 
 export const ItemCollectionModel = {
   fetch: fetch,
+  getUrl: function(ic_id, relativeUrl=false){
+    if (relativeUrl) {
+      return '/ic/' + ic_id;
+    }
+    return IndexModel.url_root + '/ic/' + ic_id;
```

Comment:
This assumes `url_root` does not have a trailing slash. It's better to assume it *always* has a trailing slash.